<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-128953572-4"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-128953572-4');
  </script>


  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Max W. Shen</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="icon" type="image/png" href="images/favicon.png">

</head>
<body style="background-color:#f9f7ee">

  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="row", style="margin-top: 5%">

      <!-- Header -->
      <div class="row">
        <!-- Photo -->
        <div class="three columns", style="margin-left: 9%">
            <h4>Max W. Shen</h4>
            <img src="images/portrait-outdoor-highres-cropped-v2-compressed-c-contrastv2.jpg", style="height: 220px", class="ImageBorder"></img>
        </div>

        <!-- Hello -->
        <div class="five columns", style="margin-top: 6%; margin-left: 2%">
          <h6>
            Hi, I am a Principal Scientist at Genentech, Prescient Frontiers Research.
            I research machine learning and biology. Forbes 30 Under 30 Science. 
          </h6>
        </div>

        <!-- Social media & contact -->
        <div class="five columns", style="margin-top: 0.5%">
          <!-- Column 1 -->
          <div class="eight columns">
            <p>
              <img src="images/email.png", style="height: 16px"></img> maxwshen [at] gmail.com <br>
              <!-- <a href="mailto:maxwshen@gmail.com"> <img src="images/email.png", style="height: 16px"></img> maxwshen@gmail.com</a> <br> -->
              <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ"> <img src="images/scholar.png", style="height: 16px"></img> Google Scholar</a> <br>
              <img src="images/location.png", style="height: 16px"></img> SF Bay Area, CA, USA
            </p>
          </div>
          <!-- Column 2 -->
          <div class="four columns">
            <p>
              <a href="assets/cv_maxwshen_210329.pdf" download> <img src="images/document.png", style="height: 16px"></img> CV </a> <br>
              <a href="https://github.com/maxwshen"> <img src="images/GitHub-Mark-120px-plus.png", style="height: 16px"></img> GitHub</a> <br>
              <!-- <a href="https://www.linkedin.com/in/maxwshen/"> <img src="images/LI-In-Bug.png", style="height: 16px"></img> LinkedIn</a> <br> -->
              <a href="https://twitter.com/maxwshen"> <img src="images/Twitter_Logo_Blue.png", style="height: 16px"></img> Twitter</a> <br>
            </p>
          </div>
        </div>
      </div>

      <div class="nine columns">

        <!-- <div class="four columns"> -->

        <!-- </div> -->

        <!-- <div class="eight columns"> -->
        <h6>
          <!-- As a computationalist at heart, I believe in interdisciplinarity, getting one's feet wet, and that better solutions to real-world problems arise by marrying each problem's unique structure with thoughtful modeling and inference design. -->

        </h6>
        <!-- </div> -->

        <h6>
          <!-- How can we get the best of deep learning and human scientific knowledge? How can we build models with improved generalization, and can build models that generalize in systematic ways that we can understand? I am interested in exploring these questions with representation learning of causal structure, expanding the expressivity of deep models with tools like the implicit function theorem, controlled generative modeling, curvature-aware optimization, and invariant causal prediction and invariant risk minimization. -->
          <!-- Science is an interesting domain for ML we often have more truth than quality data, unlike ML for general intelligence domains like vision and language. -->
        </h6>

        <h6>
          I've led research published in <i>Nature</i>, <i>Cell</i> (with a front cover featuring my pixel artwork), <i>Nature Biotechnology</i>, <i>ICML</i>, and invited for talks at NeurIPS workshops on ML for biology. I did my PhD at the Broad Institute and MIT's Computer Science & Artificial Intelligence Laboratory with David R. Liu, Aviv Regev, and formerly, David Gifford. Before that, I graduated summa cum laude from U.C. San Diego.
        </h6>

        <h6>
          Service: Area chair and co-organizer of ICLR Workshop on Machine Learning for Drug Discovery, 2022 & 2023.
        </h6>

        <!-- Social media -->
        <div class="row">


        </div>

        <!-- Publications -->
        <div>
          <hr>

          <h5>Publications</h5>

          <p>
            Towards Understanding and Improving GFlowNet Training<br>
            <b>Max W Shen</b>, Emmanuel Bengio, Ehsan Hajiramezanali, Andreas Loukas, Kyunghyun Cho, Tommaso Biancalani<br>
            <i>Accepted at ICML 2023</i><br>
          </p>

          <p>
            Deep Fitness Inference for Drug Discovery with Directed Evolution<br>
            Nathaniel Diamant, Christina Helmling, Kangway Chuang, Christian Cunningham, Tommaso Biancalani, Gabriele Scalia*, <b>Max W Shen* </b> <br>
            <i>Invited talk at NeurIPS 2022, Learning Meaningful Representations of Life Workshop</i><br>
          </p>

          <p>
            <a href="https://www.nature.com/articles/s41589-021-00876-6">Reconstruction of evolving gene variants and fitness from short sequencing reads</a> <br>
            <b>Max W Shen</b>, Kevin T. Zhao, David R. Liu<br>
            <i>Invited talk at NeurIPS 2020, Learning Meaningful Representations of Life Workshop</i><br>
            <i>Nature Chemical Biology, 2021.</i>
          </p>

          <p>
            <img src="images/cell_cover.jpg", style="height: 200px", class="ImageBorder"></img> <img src="images/behive-model.png", style="height: 200px", class="ImageBorder"></img>  <br>
            <a href="https://doi.org/10.1016/j.cell.2020.05.037">Determinants of Base Editing Outcomes from Target Library Analysis and Machine Learning</a>  <br>
            <b>Max W Shen*</b>, Mandana Arbab*, Beverly Mok, Christopher Wilson, Żaneta Matuszek, Christopher A. Cassa, David R. Liu<br>
            <i>Cell</i>, 2020. My artwork was featured on the cover of the July 23, 2020 issue!<br>
            [Co-first author reordering approved by all co-first authors]<br>
            [<a href="https://github.com/maxwshen/be_predict_bystander">Code</a>] [<a href="https://www.crisprbehive.design">Interactive web app</a>] <br>
          </p>
          <p>
            <img src="images/fig-coverplus.PNG", style="height: 200px", class="ImageBorder"></img> <img src="images/fig-indel-len-frozen.png", style="height: 200px", class="ImageBorder"></img>  <br>
            <a href="https://rdcu.be/baVVn">Predictable and precise template-free CRISPR editing of pathogenic variants</a> <br>
            <b>Max W Shen*</b>, Mandana Arbab*, Jonathan Y Hsu, Daniel Worstell, Sannie J Culbertson, Olga Krabbe, Christopher A Cassa, David R Liu, David K Gifford, Richard I Sherwood<br>
            <i>Nature</i>, 2018<br>
            [<a href="https://github.com/maxwshen/inDelphi-model">Code</a>] [<a href="https://www.crisprindelphi.design">Interactive web app</a>] [Press feature by <a href="https://plotly.com/resources/indelphi/">Dash plotly for data visualization</a>] <br>
          </p>

          <p>
            <a href = "https://www.nature.com/articles/s41587-021-00938-z">Efficient C•G-to-G•C base editors developed using CRISPRi screens, target-library analysis and machine learning</a><br>
            Luke W Koblan*, <b>Max W Shen*</b>, Mandana Arbab*, Jeffrey A Hussmann, Andrew V Anzalone, Jordan L Doman, Gregory A Newby, Dian Yang, Beverly Mok, Joseph M Replogle, Albert Xu, Tyler A Sisley, Jonathan S Weissman, Britt Adamson, David R Liu<br>
            [Co-first author reordering approved by all co-first authors]<br>
            <i>Nature Biotechnology</i>, 2021.<br>
          </p>

          <p>
            <a href = "https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008789">Small molecule inhibition of ATM kinase increases CRISPR-Cas9 1-bp insertion frequency</a><br>
            Heysol C Bermudez-Cabrera, Sannie Culbertson, Sammy Barkal, Benjamin Holmes, <b>Max W Shen</b>, Sophia Zhang, David K Gifford, Richard I Sherwood<br>
            <i>Nature Communications</i>, 2021.<br>
          </p>

          <p>
            <a href = "https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008789">Detection of gene cis-regulatory element perturbations in single cell transcriptomes</a><br>
            Grace H.T. Yeo, Oscar Juez, Qing Chen, Budhaditya Banerjee, Lendy Chu, <b>Max W Shen</b>, May Sabry, Ive Logister, Richard I Sherwood, David K Gifford<br>
            <i>PLOS Computational Biology</i>, 2021.<br>
          </p>

          <p>
            <a href = "https://www.nature.com/articles/s41467-021-21337-7">PrimeDesign software for rapid and simplified design of prime editing guide RNAs</a><br>
            Jonathan Y Hsu, Julian Grünewald, Regan Szalay, Justine Shih, Andrew V Anzalone, Kin Chung Lam, <b>Max W Shen</b>, Karl Petri, David R Liu, Keith Joung, Luca Pinello<br>
            <i>Nature Communications</i>, 2021.<br>
          </p>

          <p>
            <a href = "https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008605">Machine learning based CRISPR gRNA design for therapeutic exon skipping</a><br>
            Wilson Louie, <b>Max W Shen</b>, Zakir Tahiry, Sophia Zhang, Daniel Worstell, Christopher A Cassa, Richard I Sherwood<br>
            <i>PLOS Computational Biology</i>, 2021.<br>
          </p>

          <p>
            <a href = "https://pubmed.ncbi.nlm.nih.gov/33238122/">Comprehensive Mapping of Key Regulatory Networks that Drive Oncogene Expression</a><br>
            Lin Lin, Benjamin Holmes, <b>Max W. Shen</b>, Darnell Kammeron, Niels Geijsen, David K Giffiord, Richard I Sherwood<br>
            <i>Cell Reports</i>, 2020.<br>
          </p>

          <p>
            <a href = "https://www.nature.com/articles/s41587-020-0412-8">Continuous evolution of SpCas9 variants compatible with non-G PAMs</a><br>
            Shannon M Miller*, Tina Wang*, Peyton B Randolph, Mandana Arbab, <b>Max W Shen</b>, Tony P Huang, Zaneta Matuszek, Gregory A Newby, Holly A Rees, David R Liu <br>
            <i>Nature Biotechnology</i>, 2020<br>
            [<a href="https://github.com/maxwshen/pamvar-processing">Code</a>]  <br>
          </p>

          <p>
            <a href="https://www.pnas.org/content/113/52/E8396.short">Assembly of long error-prone reads using de Bruijn graphs</a> <br>
            Yu Lin*, Jeffrey Yuan*, Mikhail Kolmogorov, <b>Max W Shen</b>, Mark Chaisson, Pavel A Pevzner <br>
            <i>Proceedings of the National Academy of Sciences</i>, 2016<br>
          </p>

          <p>
            <a href="https://doi.org/10.1093/bioinformatics/btw493">plasmidSPAdes: assembling plasmids from whole genome sequencing data</a> <br>
            Dmitry Antipov, Nolan Hartwick, <b>Max W Shen</b>, Mikahil Raiko, Alla Lapidus, Pavel A. Pevzner <br>
            <i>Bioinformatics</i>, 2016<br>
          </p>

          <p>
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811913009580">MEG source imaging method using fast L1 minimum-norm and its applications to signals with brain noise and human resting-state source amplitude images</a> <br>
            Ming-Xiong Huang, Charles W Huang, Ashley Robb, AnneMarie Angeles, Sharon L Nichols, Dewleen G Baker, Tao Song, Deborah L Harrington, Rebecca J Theilmann, Ramesh Srinivasan, David Heister, Mithun Diwakar, Jose M Canive, J Christopher Edgar, Yu-Han Chen, Zhengwei Ji, <b>Max W Shen</b>, Fady El-Gabalawy, Michael Levy, Robert McLay, Jennifer Webb-Murphy, Thomas T Liu, Angela Drake, Roland R Lee <br>
            <i>Neuroimage</i>, 2014<br>
          </p>
        </div>

        <h5>Technical writings</h5>

        <p>
          Hierarchically branched diffusion models for efficient and interpretable multi-class conditional generation <br>
          Alex Tseng, Tommaso Biancalani, <b>Max W Shen</b>, Gabriele Scalia <br>
          <i>2023</i>
        </p>

        <p>
          Conditional Diffusion with Less Explicit Guidance via Model Predictive Control <br>
          <b>Max W Shen</b>, Ehsan Hajiramezanali, Gabriele Scalia, Alex Tseng, Nathaniel Diamant, Tommaso Biancalani <br>
          <i>arXiv preprint, 2022</i>
        </p>

        <p>
          <a href="https://arxiv.org/abs/2202.05302">Trust in AI: Interpretability is not necessary or sufficient, while black-box interaction is necessary and sufficient</a> <br>
          <b>Max W Shen</b><br>
          <i>arXiv preprint, 2022.</i>
        </p>

        <p>
          Stochastic gradient descent implicitly regularizes towards invariant models <br>
          <b>Max W Shen</b>, Anastasiya Belyaeva<br>
          <i>2021.</i>
        </p>

        <!-- Talks -->
        <!-- <div>
          <hr>
          <h5>Recognition</h5>
          <p>
            Invited talk at Models, Inference and Algorithms at the Broad Institute. Spring 2020 — delayed. <br>
            Invited talk at Synthego, Jan. 2019. <br>
            Selected talk (top 15%) at Biology of Genomes, Cold Spring Harbor, May 2018. <br>
            NSF Graduate Research Fellowship (top 15%), 2015. <br>
          </p>
        </div> -->

        <!-- Teaching -->
        <div>
          <hr>
          <h5>Teaching</h5>
          <p>
             <a href="https://github.com/maxwshen/iap-cidl">Causal Inference & Deep Learning</a> <br>
             MIT independent activites period, Jan. 2018 <br>
             <b>Max W. Shen</b>, Fredrik Johansson <br>
              ○ Prepared and co-taught a short graduate-level class with 4 sessions and 6 total h. Typical attendance: 20 students. 
          </p>
          <p>
             <a href="https://github.com/maxwshen/iap-appbml">Applied Probabilistic Programming & Bayesian Machine Learning</a> <br>
             MIT independent activites period, Jan. 2017 <br>
             <b>Max W. Shen</b>, Alvin Shi, Carles Boix <br>
             ○ Prepared and co-taught a short upper-division class with 6 sessions and 9 total h. First class attendance: 100 students, typical attendance: 25 students. 
          </p>
        </div>

        <!-- Skills -->
        <!-- <div>
          <hr>
          <h5>Skills</h5>
          <p>
            <u>Applied Machine Learning</u>: My work customizes powerful, modern deep models to leverage the unique structure within each real world problem. I have designed <a href="https://github.com/maxwshen/be_predict_bystander">deep conditional autoregressive models to model base editing outcomes</a>, and <a href="https://github.com/maxwshen/inDelphi-model">jointly-trained multitask sister deep networks to accurately learn a particularly noisy subset of CRISPR editing activity</a>. I have taught classes to MIT undergrad and graduate students on <a href="https://github.com/maxwshen/iap-appbml">Bayesian modeling</a>, <a href="https://github.com/maxwshen/iap-cidl">deep learning, and causal inference</a>. I am proficient in pytorch, and enjoy keeping up with modern toolkits: see <a href="https://github.com/maxwshen/latent-multiGP-inference">my low-level integration between pyro and gpytorch to learn 10 latent Gaussian processes on time series data in a larger probabilistic model with stochastic variational inference</a>.
          </p>

          <p>
            <u>Software Engineering</u>: See my <a href="https://github.com/maxwshen">GitHub</a>. I completed software engineering internships at Qualcomm Korea (2013) and Illumina (2014). Python is my language of choice, though I have previously worked in C++. Summa cum laude B.S. in Computer Science with a specialization in bioinformatics (2011-2015). 
          </p>

          <p>
            <u>Data Visualization</u>: My interactive <a href="https://www.crisprindelphi.design">web</a> <a href="https://www.crisprbehive.design">apps</a> have received <a href="https://plotly.com/resources/indelphi/">press attention from Dash plotly</a>. I am proficient in Adobe Illustrator, Photoshop, Premiere Pro, and After Effects, and use matplotlib, pandas, and seaborn everyday. I am also proficient in html, css, and dash plotly.
          </p>

          <p>
            <u>Communication and Collaboration</u>: My Ph.D. has featured extensive collaboration with wet lab experimentalists, including <a href="http://sherwoodlab.bwh.harvard.edu/">Richard I. Sherwood</a> and <a href="https://www.researchgate.net/profile/Mandana_Arbab">Mandana Arbab</a>. I have completed a 40 h course on conflict management and mediation that has substantially impacted my life.
          </p>

          <p>
            <u>Management</u>: For two years in undergrad, I managed 9 teams with a total of ~100 students to host a regional urban-style (hip hop) dance competition with ~2,000 audience members with revenue and expenses of $35,000/year. Each team had ~10 students and 2 team leaders, and all team leaders were overseen by me and one other co-leader. This provided substantial public speaking experience, as I led dozens of meetings speaking to and motivating our team of 100 students.
          </p>

        </div> -->

        <!-- Footer -->
        <div class="row">
          <hr>
          <div class="seven columns">
            <p>
              Max W. Shen is a Principal Scientist at Genentech.
            </p>
            <p style="color: gray">
              This website was designed with <a href="http://getskeleton.com/">Skeleton CSS</a>.

            </p>
          </div>

          <div class="three columns">
            <p>
              <a href="mailto:maxwshen@gmail.com">maxwshen@gmail.com</a> <br>
              <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ">Google Scholar</a> <br>
              <a href="https://www.linkedin.com/in/maxwshen/">LinkedIn</a> <br> 
              <a href="https://github.com/maxwshen">GitHub</a> <br>
              <a href="https://twitter.com/maxwshen">Twitter</a>  <br>
            </p>
          </div>

        </div>

      </div>
    </div>
  </div>

<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
</body>
</html>
