<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Max W. Shen</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="icon" type="image/png" href="images/favicon.png">

</head>
<body style="background-color:#f9f7ee">

  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="row">
      <div class="nine columns" style="margin-top: 10%; margin-left: 10%">
        <h4>Max W. Shen</h4>
        <h6>
          My research uses applied machine learning and statistical methods for fundamental scientific discovery and high-impact applications. Some interests include genome editing, directed evolution, and the intersection between deep learning, causal inference, and Bayesian models. As a computationalist at heart, I believe in interdisciplinarity, getting one's feet wet, and that better solutions to real-world problems arise by marrying each problem's unique structure with thoughtful modeling and inference design.
        </h6>
        <h6>
          During my Ph.D. at MIT, I have published co-first author papers in <i>Nature</i> and <i>Cell</i>. Currently, I am completing my Ph.D. and am actively looking for opportunities for late 2020 or early 2021.
        </h6>
        <h6>
          I performed my Ph.D. work with <a href = "https://liugroup.us/">David R. Liu</a> and <a href = "https://biology.mit.edu/profile/aviv-regev/">Aviv Regev</a> at the Broad Institute from 2018-now. Before, I started my Ph.D in Computational & Systems Biology with <a href = "https://cgs.csail.mit.edu/">David Gifford</a> in the MIT Computer Science & Artificial Intelligence Laboratory from 2016-2018 while collaborating closely with <a href = "http://sherwoodlab.bwh.harvard.edu/">Richard I. Sherwood</a>. Before that, I graduated summa cum laude with a B.S. in Computer Science with a specialization in bioinformatics from U.C. San Diego and did research with <a href = "https://bioalgorithms.ucsd.edu/">Pavel A. Pevzner</a> and <a href = "https://profiles.ucsd.edu/mingxiong.huang">Mingxiong Huang</a>.
        </h6>


        <p>
          <a href="https://twitter.com/maxwshen">Twitter</a>  /  
          <a href="https://www.linkedin.com/in/maxwshen/">LinkedIn</a> / 
          <a href="https://github.com/maxwshen">GitHub</a> / 
          <a href="mailto:maxwshen@gmail.com">maxwshen@gmail.com</a> / 
          <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ">Google Scholar</a> / 
          Currently based in Cambridge, MA, USA
        </p>

        <!-- Publications -->
        <div>
          <hr>
          <h5>Publications</h5>
          <p>
            <a href="https://doi.org/10.1016/j.cell.2020.05.037">Determinants of Base Editing Outcomes from Target Library Analysis and Machine Learning</a>  <br>
            Mandana Arbab*, <b>Max W Shen*</b>, Beverly Mok, Christopher Wilson, Żaneta Matuszek, Christopher A. Cassa, David R. Liu<br>
            <i>Cell</i>, 2020<br>
            [<a href="https://github.com/maxwshen/be_predict_bystander">Code</a>] [<a href="https://www.crisprbehive.design">Interactive web app</a>] <br>
          </p>
          <p>
            <a href="https://rdcu.be/baVVn">Predictable and precise template-free CRISPR editing of pathogenic variants</a> <br>
            <b>Max W Shen*</b>, Mandana Arbab*, Jonathan Y Hsu, Daniel Worstell, Sannie J Culbertson, Olga Krabbe, Christopher A Cassa, David R Liu, David K Gifford, Richard I Sherwood<br>
            <i>Nature</i>, 2018<br>
            [<a href="https://github.com/maxwshen/inDelphi-model">Code</a>] [<a href="https://www.crisprindelphi.design">Interactive web app</a>] [Press feature by <a href="https://plotly.com/resources/indelphi/">Dash plotly for data visualization</a>] <br>
          </p>

          <p>
            <a href = "https://www.nature.com/articles/s41587-020-0412-8">Continuous evolution of SpCas9 variants compatible with non-G PAMs</a><br>
            Shannon M Miller*, Tina Wang*, Peyton B Randolph, Mandana Arbab, <b>Max W Shen</b>, Tony P Huang, Zaneta Matuszek, Gregory A Newby, Holly A Rees, David R Liu <br>
            <i>Nature Biotechnology</i>, 2020<br>
            [<a href="https://github.com/maxwshen/pamvar-processing">Code</a>]  <br>
          </p>

          <p>
            <a href="https://www.pnas.org/content/113/52/E8396.short">Assembly of long error-prone reads using de Bruijn graphs</a> <br>
            Yu Lin*, Jeffrey Yuan*, Mikhail Kolmogorov, <b>Max W Shen</b>, Mark Chaisson, Pavel A Pevzner <br>
            <i>Proceedings of the National Academy of Sciences</i>, 2016<br>
          </p>

          <p>
            <a href="https://doi.org/10.1093/bioinformatics/btw493">plasmidSPAdes: assembling plasmids from whole genome sequencing data</a> <br>
            Dmitry Antipov, Nolan Hartwick, <b>Max W Shen</b>, Mikahil Raiko, Alla Lapidus, Pavel A. Pevzner <br>
            <i>Bioinformatics</i>, 2016<br>
          </p>

          <p>
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811913009580">MEG source imaging method using fast L1 minimum-norm and its applications to signals with brain noise and human resting-state source amplitude images</a> <br>
            Ming-Xiong Huang, Charles W Huang, Ashley Robb, AnneMarie Angeles, Sharon L Nichols, Dewleen G Baker, Tao Song, Deborah L Harrington, Rebecca J Theilmann, Ramesh Srinivasan, David Heister, Mithun Diwakar, Jose M Canive, J Christopher Edgar, Yu-Han Chen, Zhengwei Ji, <b>Max W Shen</b>, Fady El-Gabalawy, Michael Levy, Robert McLay, Jennifer Webb-Murphy, Thomas T Liu, Angela Drake, Roland R Lee <br>
            <i>Neuroimage</i>, 2014<br>
          </p>
        </div>

        <!-- Footer -->
        <div>
          <hr>
          <div class="seven columns">
            <p>
              Max W. Shen is a Ph.D. candidate at MIT, working on applied machine learning and statistical methods for fundamental scientific discovery and high-impact applications.
            </p>
            <p style="color: gray">
              This website was designed with <a href="http://getskeleton.com/">Skeleton CSS</a>.

            </p>
          </div>

          <div class="one columns">
            <p>
              <a href="https://twitter.com/maxwshen">Twitter</a>  <br>
              <a href="https://www.linkedin.com/in/maxwshen/">LinkedIn</a> <br> 
              <a href="https://github.com/maxwshen">GitHub</a> <br>
              <a href="mailto:maxwshen@gmail.com">maxwshen@gmail.com</a> <br>
              <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ">Google_Scholar</a> <br>
            </p>
          </div>

        </div>

      </div>
    </div>
  </div>

<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
</body>
</html>
